.github/workflows/build_and_test.yml

```diff
@@ -528,6 +515,24 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 8
+    - name: Cache TPC-DS generated data
+      id: cache-tpcds-sf-1
+      uses: actions/cache@v2
+      with:
+        path: ./tpcds-sf-1
+        key: tpcds-${{ hashFiles('sql/core/src/test/scala/org/apache/spark/sql/TPCDSSchema.scala') }}
```

Comment:
Is this sufficient?

Reply:
yes, I think it is for now since this PR targets at schema definition changes.